tests/unit/ActionChip.spec.ts

```diff
@@ -3,15 +3,14 @@ import Vuetify from 'vuetify'
 import Affix from 'vue-affix'
 import { shallowMount } from '@vue/test-utils'
 import { ActionChip } from '@/components/action-chip'
-import { ActionTypes } from '@bcrs-shared-components/enums'
 
 Vue.use(Vuetify)
 Vue.use(Affix)
 
 const vuetify = new Vuetify({})
 
 describe('ActionChip', () => {
-  const wrapperFactory = (action: ActionTypes, editedLabel: string = null) => {
```

Comment:
Line 6 generated an error, but since we don't want types in the unit tests anyways, this change fixed both.